matchzoo/models/mvlstm.py

```diff
@@ -0,0 +1,70 @@
+"""An implementation of MVLSTM Model."""
+import typing
+import logging
+
+import keras
+import keras.backend as K
+
+from matchzoo import engine
+
+logger = logging.getLogger(__name__)
+
+
+class MVLSTM(engine.BaseModel):
+    """
+    MVLSTM Model.
+
+    Examples:
+        >>> model = MVLSTM()
+        >>> model.params['mlp_num_layers']=2
```

Comment:
`num_mlp_layers`

Reply:
I guess he wanted to be consistent with those params in base model, so I think it's fine.